particle.js

```diff
@@ -89,12 +90,20 @@ const entryPrepend = entry => ({
  * @param {Object} app.shared - Shared webpack config common to dev and prod
  * @param {Object} app.dev - Webpack config unique to prod
  * @param {Object} app.prod - Webpack config unique to prod
+ * @param {string} designSystemPath - Webpack config per-design system
  * @param {Object} options - Compile options
  * @param {('hot'|'extract')} options.cssMode - The method of handling CSS output
- * @param {string} options.entry - The main entry point to prepend polyfills
+ * @param {string} [options.entry] - The main entry point to prepend polyfills
  * @returns {*} - Fully merged and customized webpack config
  */
-const particle = (app, options) => {
+const particle = (app, designSystemPath, options) => {
+  // Dynamically pull in design system config. Must be named webpack.config.js
```

Comment:
this comment about name req should be in all caps or flagged somehow. definitely triple underscores in docs

Reply:
Will make sure this is captured in docs, but for now copying a design system folder should work.